src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala

```diff
@@ -98,7 +98,7 @@ class GraphAuthDAO extends AuthDAO with GraphDAO {
         userPipeline(db, user).as("vtx").in()
           .loop(
             "vtx", //start point of loop
-            invert(isVertexOfClass(VertexSchema.Workspace)), //loop condition: stop walking the path once we've found a workspace node
+            or(isVertexOfClass(VertexSchema.Group), isVertexOfClass(VertexSchema.Map)), //loop condition: walk while vertex is on an acl path (group or a list which looks like map)
```

Comment:
submission now has an edge to user so we need to be more careful in the path we walk from user up to workspace. instead of following anything that isn't a workspace we follow only group and map vertexes.